testutil/ids.go

```diff
@@ -24,6 +27,14 @@ func AccAddress(t testing.TB) sdk.AccAddress {
 	return info.GetAddress()
 }
 
+// DeterministicAccAddress takes a seed []byte paramater for deterministic
+// private key generation.
+func DeterministicAccAddress(t testing.TB, seed []byte) sdk.AccAddress {
+	t.Helper()
+	privKey := ed25519.GenPrivKeyFromSecret(seed)
+	return sdk.AccAddress(privKey.PubKey().Address())
```

Comment:
I very much prefer this method to creating a keyring - would be good to re-implement `AccAddress` like this (sans asking the caller to pass a seed in).

Reply:
I added a dedicated method to generate it without the seed. I think it'll be useful in other places.